gatling-http/src/test/scala/io/gatling/http/HttpCompileTest.scala

```diff
@@ -1,284 +1,299 @@
+    .exec(http("Ajout au panier").get("/").check(regex(session => """<input id="(.*)" type="text" value="(.*)" />""").ofType[(String, String)] saveAs ("input")))
+    .pause(pause1)
+    .exec(polling.every(10.seconds).exec(http("poll").get("/foo")))
+    .exec(polling.pollerName("poll").every(10.seconds).exec(http("poll").get("/foo")))
+    .exec(polling.pollerName("poll").stop)
+    .exec(polling.stop)
+    .rendezVous(100)
+    .exec(flushSessionCookies)
+    .pause(pause4)
+    .exec(addCookie(Cookie("foo", "bar").withDomain("/")))
+    .pace(5)
+    .pace(5 seconds)
+    .pace("${foo}")
+    .pace(5, 10)
+    .pace("${foo}", "${bar}")
+    .doSwitch("${foo}")(
+      "a" -> exec(http("a").get("/")),
+      "b" -> exec(http("b").get("/"))
+    )
+    .doSwitchOrElse("${foo}")(
+      "a" -> exec(http("a").get("/")),
+      "b" -> exec(http("b").get("/")) //
+    )(exec(http("else").get("/")))
+
+  val inject1 = nothingFor(10 milliseconds)
+  val inject2 = rampUsers(10).over(10 minutes)
+  val inject3 = constantUsersPerSec(10).during(1 minute)
+  val inject4 = atOnceUsers(100)
+  val inject5 = rampUsersPerSec(10) to (20) during (10 minutes)
+  val inject6 = splitUsers(1000).into(rampUsers(10) over (10 seconds)).separatedBy(10 seconds)
+  val inject7 = splitUsers(1000).into(rampUsers(10) over (10 seconds)).separatedBy(atOnceUsers(30))
+  val inject8 = heavisideUsers(1000) over (20 seconds)
+
+  val injectionSeq = Vector(1, 2, 4, 8).map(x => rampUsers(x * 100) over (5 seconds))
+  setUp(
+    lambdaUser.inject(inject1),
+    lambdaUser.inject(injectionSeq: _*),
+    lambdaUser.inject(inject1, inject2).throttle(jumpToRps(20), reachRps(40) in (10 seconds), holdFor(30 seconds))
+  )
+    .protocols(httpProtocol)
+    .pauses(uniformPausesPlusOrMinusPercentage(1))
+    .disablePauses
+    .constantPauses
+    .exponentialPauses
+    .uniformPauses(1.5)
+    .uniformPauses(1337 seconds)
+    .assertions(
+      global.responseTime.mean.lessThan(50),
+      global.responseTime.max.between(50, 500),
+      global.successfulRequests.count.greaterThan(1500),
+      global.allRequests.percent.is(100),
+      forAll.failedRequests.percent.is(0),
+      forAll.responseTime.max.is(100),
+      details("Users" / "Search" / "Index page").responseTime.mean.greaterThan(0),
+      details("Admins" / "Create").failedRequests.percent.lessThan(90),
+      details("request_9").requestsPerSec.greaterThan(10)
+    )
+    .throttle(jumpToRps(20), reachRps(40) in (10 seconds), holdFor(30 seconds))
+    // Applies on the setup
+    .constantPauses
+    .disablePauses
+    .exponentialPauses
+    .uniformPauses(1.5)
+    .uniformPauses(1337 seconds)
+
+  import io.gatling.http.check.HttpCheck
+  import io.gatling.core.check.ConditionalCheck
+  import io.gatling.core.session.Session
+  import io.gatling.commons.validation.Success
+
+  val JSON_CONTENT_TYPE_VALUE: String = "application/json"

```

Comment:
Already available as `HttpHeaderValues.ApplicationJson`.
Plus, you use the Java convention for constants, not the Scala one :)